cli/internal/cmd/log.go

```diff
@@ -7,25 +7,25 @@ SPDX-License-Identifier: AGPL-3.0-only
 package cmd
 
 import (
-	"github.com/edgelesssys/constellation/v2/internal/logger"
+	"log/slog"
+	"os"
+
 	"github.com/spf13/cobra"
-	"go.uber.org/zap/zapcore"
 )
 
 type debugLog interface {
-	Debugf(format string, args ...any)
-	Sync()
+	Debug(format string, args ...any)
```

Comment:
```suggestion
	Debug(msg string, args ...any)
```
please also find all other places where "format string" needs to be replaced with "msg string"